plugins/dynamic-plugins-info-backend/src/service/router.ts

```diff
@@ -0,0 +1,37 @@
+import { errorHandler } from '@backstage/backend-common';
+import {
+  PluginManager,
+  BaseDynamicPlugin,
+} from '@backstage/backend-plugin-manager';
+import express from 'express';
+import Router from 'express-promise-router';
+import { Logger } from 'winston';
+
+export interface RouterOptions {
+  logger: Logger;
+  pluginManager: PluginManager;
+}
+
+export async function createRouter(
+  options: RouterOptions,
+): Promise<express.Router> {
+  const { pluginManager } = options;
+
+  const router = Router();
+  router.use(express.json());
+
+  const plugins = pluginManager.plugins;
+  const dynamicPlugins = plugins.map(p => {
+    // Remove the installer details for the dynamic backend plugins
+    if (p.platform === 'node') {
+      const { installer, ...rest } = p;
+      return rest as BaseDynamicPlugin;
+    }
+    return p as BaseDynamicPlugin;
+  });
+  router.get('/dynamic-plugins', (_, response) => {
```

Comment:
Could we name this endpoint: `loaded-plugins` ?
That would leave room for other endpoints in the future like: the new plugins that should be installed / removed after the restart, the failure to install some plugins, etc ...

Reply:
sure